hugegraph-llm/src/tests/operators/hugegraph_op/test_fetch_graph_data.py

```
@@ -0,0 +1,153 @@
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import unittest
+from unittest.mock import MagicMock, patch
```

Ruff (0.11.9)

`unittest.mock.patch` imported but unused

Remove unused import: `unittest.mock.patch`

(F401)